<?xml version="1.0"?>
<launch>
    <rosparam command="load" file = "$(find autonomy)/params/autonomy_simulator.yaml" ns="autonomy"/>

   <!--
    <node name="transit_node" pkg="autonomy" type="transit_node.py" launch-prefix="xterm -e"/>
   -->
   
   <node name="simulator_node" pkg="autonomy" type="simulator_node.py"/>
</launch>
